src/parser/Makefile.in

```diff
@@ -420,7 +420,7 @@ mostlyclean-compile:
 	-rm -f *.$(OBJEXT)
 	-rm -f $(top_srcdir)/src/utils/error/*.$(OBJEXT)
 	-rm -f $(top_srcdir)/src/utils/mmgr/*.$(OBJEXT)
-	-rm -f ../age/*.$(OBJEXT)
+	-rm -f $(top_srcdir)/age/*.$(OBJEXT)
```

Comment:
IMO `$(top_srcdir)/age/` should be `$(top_srcdir)/src/age/`. need to update all occurrences in MakeFiles